src/plugins/discover/public/application/view_components/canvas/discover_chart_container.tsx

```diff
@@ -16,27 +16,26 @@ export const DiscoverChartContainer = ({ hits, bucketInterval, chartData }: Sear
   const { uiSettings, data } = services;
   const { indexPattern, savedSearch } = useDiscoverContext();
 
-  const timeField = indexPattern?.timeFieldName;
-
-  if (!hits || !bucketInterval || !chartData) {
-    // TODO: handle better
-    return null;
-  }
```

Comment:
Just double checking. We are good with not having `bucketInterval` or `chartData`?